### DISCUSSION TITLE
Laminar Discussions – Github Test Run

### DISCUSSION THREAD

**Discussion by raquo** [raquo/Laminar#80] [General]
Hey everyone, up to this date we've been chatting in [gitter](https://gitter.im/Laminar_/Lobby).

I only chose gitter because this is what the rest of the Scala ecosystem has been using, but I'm not a big fan of it. Gitter content is entirely unstructured, hard to search, and the chat interface itself leaves a lot to be desired.

So, let's try github discussions instead, and see how it goes.

**Comment by raquo** [comment]
Ha, we `auto_increment`-ed our way to issue # 79 the hard way, with just issues & PRs, but I guess discussion IDs are coming from the same sequence (this one is #80). Oh well.

**Reply by yurique** [reply]
interesting :)

**Comment by Miuler** [comment]
Gitter and matrix are now the same, so it seems like a great option, in the style of the old IRC, the best thing is that it is an open protocol and different open source clients, it is closer to people.

The discussions seem interesting to me, it has another approach, it seems good as a first point of contact.

**Reply by raquo** [reply]
Yeah I think we'll end up keeping both gitter and github discussions up!

**Reply by yurique** [reply]
@Miuler is there a way to sign in in the matrix client using my gitter/github user?
